tools/create_release.py

```diff
@@ -101,13 +101,13 @@ def create_patch_zip(self):
     def create_wrap_file(self):
         self.wrap_section['wrapdb_version'] = self.version
 
-        filename = Path(self.tempdir, self.name + '.wrap')
+        filename = Path(self.tempdir, f"{self.name}.wrap")
 
         # configparser write() adds multiple trailing newlines, collapse them
         buf = io.StringIO()
         self.wrap.write(buf)
         buf.seek(0)
-        newbuf = buf.read().rstrip('\n') + '\n'
+        newbuf = f"{buf.read().rstrip('\\n')}\n"
```

Comment:
This has very bad readability. Not all string concatenation should be converted to f-strings.